src/apify/_memory_storage/__init__.py

```diff
@@ -1,3 +1,3 @@
-from .memory_storage_client import MemoryStorageClient
+from apify._memory_storage.memory_storage_client import MemoryStorageClient
```

Comment:
Please, let's leave all the `__init__.py` with relative imports, since they're importing only from their siblings.

```suggestion
from .memory_storage_client import MemoryStorageClient
```